Chore/ChoreMenu.js

```diff
@@ -3,53 +3,101 @@
         runChoreIntroduction();
     }
 
-    sendVirtualAssistantMessage('Do you wish to do a new chore or return?', 0);
-
-    let lobbyAnswer = createInput('Chore', 'Info', 'Return');
-    while (true) {
+ 
+    setVar("doingchores",true);
+ 
+    while (getVar("doingchores")==true) {
```

Comment:
== true? Unnecessary check. getVar("DoingChores", false) will suffice

Reply:
if you say so.... but which is more readable?